guideline-17.md

```diff
@@ -1,7 +1,7 @@
-**17. Respect trademarks and projects.**
+<h4>17. Plugins must respect trademarks, copyrights, and project names.</h4>
 
-The use of trademarks or other projects as the sole or initial term of a plugin slug is prohibited unless proof of legal ownership/representation can be confirmed. For example, the [WordPress Foundation has trademarked the term “WordPress”](http://wordpressfoundation.org/trademark-policy/) and it is a violation to use “wordpress” in a domain name. This policy extends to plugin slugs. 
+The use of trademarks or other projects as the sole or initial term of a plugin slug is prohibited unless proof of legal ownership/representation can be confirmed. For example, the [WordPress Foundation has trademarked the term “WordPress”](http://wordpressfoundation.org/trademark-policy/) and it is a violation to use “wordpress” in a domain name.  This policy extends to plugin slugs, and we will not permit a slug to begin with another product's term.
```

Comment:
Double space before `This policy extends ...`.

Reply:
Patched! Thank you.